.github/actions/action-maven-build/action.yml

```
@@ -0,0 +1,41 @@
+    required: true
+    type: string
+    description: path to the pom.xml
+outputs:
+  artifact-name:
+    description: "name of the artifact upload"
+    value: ${{steps.artifact-name.outputs.artifact-name}}
+runs:
+
+  using: "composite"
+  steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)